cmd/crictl/version.go

```diff
@@ -32,7 +32,7 @@ var runtimeVersionCommand = &cli.Command{
 	Usage: "Display runtime version information",
 	Action: func(c *cli.Context) error {
 		if c.NArg() != 0 {
-			cli.ShowSubcommandHelp(c)
+			return cli.ShowSubcommandHelp(c)
```

Comment:
Another strange misbehavior because right now it still tries to connect even when showing the help.